### DISCUSSION TITLE
release 2.1.1

### DISCUSSION THREAD

**Discussion by Aloren** [aerospike/spring-data-aerospike#517] [Team Posts]
Hi @rbotzer

We have found an issue with spring-data-aerospike. 
I've already updated version in master and prepared draft for the version: https://github.com/aerospike-community/spring-data-aerospike/releases

Could you please release?

**Comment by rbotzer** [comment]
Just saw this, sorry about the delay. We'll do it Monday morning.

**Comment by rbotzer** [comment]
It's pushed to maven and published.
